### DISCUSSION TITLE
Show multiple ranges in calendar

### DISCUSSION THREAD

**Discussion by louistychon** [wojtekmaj/react-calendar#641] [Q&A]
Hi, 

Thanks for this amazing work. 
I'm currently working on a room reservation tool and I would like to display several ranges on the calendar on the page of the rooms (to show when rooms are already booked). 
Is it possible to pass several values (either simple values or ranges) into a props ?

[Example of what I'd like to do](https://jsfiddle.net/uon79q2c/)

Thanks,

**Comment by wojtekmaj** [comment]
It looks like your use case falls more under `tileClassName`, `tileDisabled`, `tileContent` props :)
